lib/src/screens/send/widgets/send_card.dart

```diff
@@ -438,8 +440,8 @@ class SendCardState extends State<SendCard>
                                           Padding(
                                               padding:
                                               EdgeInsets.only(top: 5),
-                                              child: Text(
-                                                  output
+                                              child: sendViewModel.balanceViewModel.disableFiat ?
+                                              Container () : Text(output
```

Comment:
you can add the if condition here too, then it won't have to build any unused widgets (Padding, Container).

if removing the padding will affect the UI then you can replace the container with `const SizedBox()`

Reply:
fixed